R/infrastructure.R

```diff
@@ -33,6 +39,12 @@ use_testthat <- function(pkg = ".") {
     pkg = pkg
   )
 
+  if (use_catch) {
```

Comment:
Maybe it would be easier to just have this as `use_catch()` rather than an argument? `use_catch()` would call `use_testthat()` if the testing infrastructure hadn't already been set up (in the same way that `use_test()` does)